src/neuroconv/nwbconverter.py

```diff
@@ -154,12 +159,7 @@ def run_conversion(
             overwrite=overwrite,
             verbose=self.verbose,
         ) as nwbfile_out:
-            for interface_name, data_interface in self.data_interface_objects.items():
-                data_interface.run_conversion(
-                    nwbfile=nwbfile_out, metadata=metadata, **conversion_options.get(interface_name, dict())
-                )
-
-        return nwbfile_out
+            self.add_to_nwbfile(nwbfile_out, metadata, conversion_options)
```

Comment:
I see, maybe the orchestrators NWBConverter and the Pipeline do have to follow a different logic here.